css/style.css

```diff
@@ -1,103 +1,139 @@
+/* cyrillic-ext */
+@font-face {
+  font-family: 'PT Mono';
+  font-style: normal;
+  font-weight: 400;
+  src: local('PT Mono'), local('PTMono-Regular'), url(/fonts/PT_Mono_ext.woff2) format('woff2');
+  unicode-range: U+0460-052F, U+1C80-1C88, U+20B4, U+2DE0-2DFF, U+A640-A69F, U+FE2E-FE2F;
+}
+/* cyrillic */
+@font-face {
+  font-family: 'PT Mono';
+  font-style: normal;
+  font-weight: 400;
+  src: local('PT Mono'), local('PTMono-Regular'), url(PT_Mono_ext.woff2) format('woff2');
```

Comment:
maybe there must be `/fonts/` in the beginning of the `url`?

Reply:
Yep, I'll fix it tomorrow :)